admin/python/lsst/qserv/admin/qservAdmin.py

```diff
@@ -314,7 +314,7 @@ def _createDbLikeLocked(self, dbName, templateDbName):
             self._createDbLockSection(dbP)
             self._kvI.set(dbP, "READY")
         except KvException as e:
-            self._logger.error("Failed to create database '%s' like '%s', error was: %s",
+            self._logger.error("Failed to create db '%s' like '%s', error was: %s",
```

Comment:
One small trick - I prefer to use `%r` instead of `'%s'` in cases when I want to have quoted (and escaped) strings on output, frees me from the need to think whether do I need to escape quotes inside string.